Source/TestDafny/MultiBackendTest.cs

```diff
@@ -415,8 +415,11 @@ int ExpectedExitCode
     return (exitCode, outputWriter.ToString(), errorWriter.ToString());
   }
 
-  private static bool OnlyUnsupportedFeaturesErrors(IExecutableBackend backend, string output) {
+  private static bool OnlyAllowedOutputLines(IExecutableBackend backend, string output) {
     using StringReader sr = new StringReader(output);
+    if (output == "") {
+      return false;
+    }
```

Comment:
Why this case? This logic generally returns true for empty input.

Reply:
So... are you approving this PR?

Reply:
Sorry, I thought I did, no idea what happened. Approved now!